docs/application/dotnet/guides/app-management/widget-control.md

```diff
@@ -0,0 +1,151 @@
+
+# Widget Control
+
+The widget control provides information about installing and running widget applications.
+
+It also provides functions for the following:
+
+*   Sending update requests to the widget applications
+*   Retrieving details of running instance for the same package widget applications
+
+
+The main features of the `Tizen.Applications.WidgetControl` class include:
+
+-   [Getting information on widget applications](#getting_information)
+
+    For installed (but not necessarily running) widget applications, you can retrieve widget information with the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class.
+
+
+-   [Listening widget lifecycle events on widget applications](#listening_events)
+
+    For running widget applications, you can listen widget application lifecycle events with the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class.
+
+
+-   [Communicating with running widget instances](#communicating_instances)
+
+    For running widget instances, you can trigger an update event, send update data, and retrieve running widget instances detail with the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class.
+
+
+## Prerequisites
+
+To enable your application to use the widget control functionality:
+
+1.  To use the methods and properties of the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class, include the [Tizen.Applications](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.html) namespace in your application:
+
+    ```
+    using Tizen.Applications;
+    ```
+
+2.  To get information on widget application, the application has to request permission by adding the following privilege to the  `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/widget.viewer</privilege>
+    </privileges>
+    ```
+
+<a name="getting_information"></a>
+## Getting Information on Widget Applications
+
+To get information on these widget applications, follow the steps:
+
+1.  Create an instance of the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class with the ID of the widget application, as shown in the following code:
```

Comment:
I can see there are 3 "Create an instance of Tizen.Applications.WidgetControl class~". 
As every section has this list item now, how about creating a new "Create an instance" section?

For example, in case of section,
 - Create an Widget Control instance
   Create an instance of the Tizen.Applications.WidgetControl class ~
 - Get Information on Widget Applications
   Get the main application ID, package ID, and available scale lists from the control:
     _-> As we have only one list item in this case, the above and the sample code would be enough_
 - Listen Widget Lifecycle Events on Widget Applications
   To listen to the widget lifecycle events, add lifecycle listeners on the control:
     _-> As we have only one list item in this case, the above and the sample code would be enough_
 - Communicate with Running Widget Instances
   To communicate with the running widget instances, follow these ~

Reply:
Done